src/Driver/PCOV.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/*
+ * This file is part of the php-code-coverage package.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace SebastianBergmann\CodeCoverage\Driver;
+
+use SebastianBergmann\CodeCoverage\RuntimeException;
+use SebastianBergmann\CodeCoverage\Filter;
+
+/**
+ * Driver for PCOV code coverage functionality.
+ *
+ * @codeCoverageIgnore
+ */
+final class PCOV implements Driver
+{
+    /**
+     * @throws RuntimeException
```

Comment:
Not sure how this constructor should throw a `RuntimeException` - am I missing something, or the constructor itself a method body?

Reply:
Copy pasta error, will fix that, ta ...